py/picca/pk1d/postproc_pk1d.py

```diff
@@ -349,6 +382,9 @@ def variance_function(snr, amp, zero_point):
                     mean = np.mean(p1d_table[col][select])
                     # unbiased estimate: num_chunks-1
                     error = np.std(p1d_table[col][select]) / np.sqrt(num_chunks - 1)
```

Comment:
shouldn't this also be computed differently when apply_z_weights is on?

Reply:
Yes, I computed and implemented the expression now.
Also fixed a bug (missing sqrt) in the other expression when SNR weighting is on